src/Field/Group/AbstractGroupingElement.php

```diff
@@ -104,4 +104,26 @@ public function transformData (
 			$dataVisitor,
 		);
 	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public function enableValidation (
+		bool $required = true,
+		?string $regexp = null,
+		bool $allowMissingData = false,
```

Comment:
We should remove these fields from the api data in the field itself

Reply:
What exactly do you mean?

Reply:
Right now we send `"required" = true` to their API even if it isn't supported?

Reply:
I've already removed that in an earlier commit 🙌 This method now will no longer set anything directly on the `AbstractGroupingElement` and only update the nested fields.